bindings/nodejs/package.json

```diff
@@ -31,8 +31,8 @@
   },
   "scripts": {
     "artifacts": "napi artifacts",
-    "build": "napi build --platform --release",
-    "build:debug": "napi build --platform",
+    "build": "napi build --platform --release --js js-binding.js --dts js-binding.d.ts",
```

Comment:
by the way, I don't like the naming of this file. Maybe we can use `generated.js`?

Reply:
OK. No problem.